dotnet/samples/SemanticKernelMigration/README.md

```diff
@@ -1,393 +1,3 @@
-
-You will be prompted for any required environment variables if they are not already set.
-
-### Prerequisites
-
-Before you begin, ensure you have the following:
-
-- [.NET 8.0 SDK or later](https://dotnet.microsoft.com/download)
-- For Azure AI Foundry samples: Azure OpenAI service endpoint and deployment configured
-- For OpenAI samples: OpenAI API key
-- For OpenAI Assistants samples: OpenAI API key with Assistant API access
-
-### Environment Variables
-
-Set the appropriate environment variables based on the sample type you want to run:
-
-**For Azure AI Foundry projects:**
-```powershell
-$env:AZURE_FOUNDRY_PROJECT_ENDPOINT = "https://<your-project>-resource.services.ai.azure.com/api/projects/<your-project>"
-```
-
-**For OpenAI and OpenAI Assistants projects:**
-```powershell
-$env:OPENAI_API_KEY = "sk-..."
-```
-
-**For Azure OpenAI and Azure OpenAI Assistants projects:**
-```powershell
-$env:AZURE_OPENAI_ENDPOINT = "https://<your-project>.cognitiveservices.azure.com/"
-$env:AZURE_OPENAI_DEPLOYMENT_NAME = "gpt-4o" # Optional, defaults to gpt-4o
-```
-
-**Optional debug mode:**
-```powershell
-$env:AF_SHOW_ALL_DEMO_SETTING_VALUES = "Y"
-```
-
-If environment variables are not set, the demos will prompt you to enter values interactively.
-
-### Samples
-
-The migration samples are organized into three categories, each demonstrating different AI service integrations:
-
-|Category|Description|
-|---|---|
-|[AzureAIFoundry](./AzureAIFoundry/)|Azure OpenAI service integration samples|
-|[AzureOpenAI](./AzureOpenAI/)|Direct Azure OpenAI API integration samples|
-|[AzureOpenAIAssistants](./AzureOpenAIAssistants/)|Azure OpenAI Assistants API integration samples|
-|[AzureOpenAIResponses](./AzureOpenAIResponses/)|Azure OpenAI Responses API integration samples|
-|[OpenAI](./OpenAI/)|Direct OpenAI API integration samples|
-|[OpenAIAssistants](./OpenAIAssistants/)|OpenAI Assistants API integration samples|
-|[OpenAIResponses](./OpenAIResponses/)|OpenAI Responses API integration samples|
-
-## Running the samples from the console
-
-To run any migration sample, navigate to the desired sample directory:
-
-```powershell
-# Azure AI Foundry Examples
-cd "AzureAIFoundry\Step01_Basics"
-dotnet run
-
-# Azure OpenAI Examples
-cd "AzureOpenAI\Step01_Basics"
-dotnet run
-
-# OpenAI Examples
-cd "OpenAI\Step01_Basics"
-dotnet run
-
-# OpenAI Assistants Examples
-cd "OpenAIAssistants\Step01_Basics"
-dotnet run
-
-# OpenAI Responses Examples
-cd "OpenAIResponses\Step01_Basics"
-
-# Azure OpenAI Examples
-cd "AzureOpenAI\Step01_Basics"
-dotnet run
-
-# Azure OpenAI Assistants Examples
-cd "AzureOpenAIAssistants\Step01_Basics"
-dotnet run
-```
+For instructions and samples describing how to migrate from Semantic Kernel to Microsoft Agent Framework see the [Semantic Kernel Repository](https://github.com/microsoft/semantic-kernel/tree/main/dotnet/samples/AgentFrameworkMigration) page.
```

Comment:
Can this note instead be added to https://github.com/microsoft/agent-framework/blob/30fafde67196741be6cad22545c7fc123213698b/dotnet/samples/README.md and then this whole folder removed?